realtime_backend/main.py

```
@@ -0,0 +1,13 @@
+
+Keeps old path `realtime_backend/main.py` working while delegating
+implementation to `crypt_mainica_backend.main`.
+"""
+
+from crypt_mainica_backend.main import app
+
+
+if __name__ == "__main__":
+    import uvicorn
+
+    uvicorn.run("realtime_backend.main:app", host="0.0.0.0", port=8000, reload=True)
```

Ruff (0.15.9)

[error] Possible binding to all interfaces

(S104)